resources/assets/js/components/ApplicantProfile/Skills/SkillSearchResults.tsx

```diff
@@ -0,0 +1,199 @@
+import * as React from "react";
+import { defineMessages, useIntl } from "react-intl";
+import { h2ComponentAccordionAddTriggerEvent } from "@hydrogen-design-system/system/dist/import/latest/components/accordion/scripts/accordion";
+import { Skill } from "../../../models/types";
+import { getLocale, localizeFieldNonNull } from "../../../helpers/localize";
+
+const resultMessages = defineMessages({
+  resultTitle: {
+    id: "results.title",
+    defaultMessage: "Results",
+    description: "The message displayed before the number of results returned.",
+  },
+  resultEmpty: {
+    id: "results.empty",
+    defaultMessage: "Nothing found",
+    description: "The message displayed if no results returned.",
+  },
+  resultAccordionExpand: {
+    id: "results.accordion.expand",
+    defaultMessage: "Click to view...",
+    description: "The message displayed to expand the accordion.",
+  },
+  resultAccordionAdd: {
+    id: "results.accordion.add",
+    defaultMessage: "Add",
+    description: "The label for the add button.",
+  },
+  resultAccordionAdded: {
+    id: "results.accordion.added",
+    defaultMessage: "Added!",
+    description: "The message displayed if skill added.",
+  },
+});
+
+interface SkillSearchResultsProps {
+  status: string;
+  results: Array<Skill> | null;
```

Comment:
It doesn't actually expect `Array<Skill>`, it expects `Array<Skill & {isChecked?: boolean}>`. In fact, I think you should make it not optional by removing the `?`. Right now, SkillSearchBar always passes down an array of Skills with no isChecked status, and technically that's valid (but I don't think it should be).